src/harvest-rpde.js

```diff
@@ -183,6 +185,7 @@ async function harvestRPDE({
       if ((WAIT_FOR_HARVEST || VALIDATE_ONLY) && isOrdersFeed) {
         onFeedEnd();
       }
+      // TODO document: What is a 'FatalError'?
```

Comment:
Interestingly we've lost the definition of FatalError here, where it was previously slightly more well-defined:

https://github.com/openactive/openactive-test-suite/blob/d81376347a3419287ba0ee05b7fb399322c09485/packages/openactive-broker-microservice/app.js#L300

Reply:
"FatalError" seems very broker-specific though and probably shouldn't be in the library? Is it work unpicking this further?

Reply:
I now understand this, thanks! I've updated the comment to explain this and suggest decoupling it in future and have created this issue to house all these decoupling-related tasks: https://github.com/openactive/harvesting-utils/issues/9